### PR TITLE
CTCP-133 Migrate WhatDoYouWantToDoView to Twirl

### PR DESCRIPTION
**Pull request by carrielies**

### CODE REVIEW

**Review by jackheslop96 — APPROVED** [review]

File: test/views/WhatDoYouWantToDoViewSpec.scala [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ */
+
+package views
+
+import models.Availability
+import play.twirl.api.HtmlFormat
+import views.behaviours.ViewBehaviours
+import views.html.WhatDoYouWantToDoView
+
+class WhatDoYouWantToDoViewSpec extends ViewBehaviours {
```
Comment (jackheslop96) [inline test/views/WhatDoYouWantToDoViewSpec.scala:24]: Might actually make sense to split this out a bit, so have something like:

```
def applyView(availability: Availability) = injector.instanceOf[WhatDoYouWantToDoView].apply(availability, availability)(fakeRequest, messages)

"when departures and arrivals empty must" - {
    behave like ...
}

"when departures and arrivals unavailable must" - {
    val doc = parseView(applyView(Unavailable))
    behave like ...
}

"when departures and arrivals non-empty must" - {
    val doc = parseView(applyView(NonEmpty))
    behave like ...
}
```

Comment (jackheslop96) [reply test/views/WhatDoYouWantToDoViewSpec.scala:24]: Could then also have something like

```
private val availability: Availability = arbitrary[Availability].sample.value

override def view: HtmlFormat.Appendable =
    applyView(availability, availability)
```

so we know that the behave like `pageWithTitle()`, `behave like pageWithBackLink()` etc. tests aren't dependent on the availability type

Comment (carrielies) [reply test/views/WhatDoYouWantToDoViewSpec.scala:24]: Looks good, I'll do that, but I'm keeping Arrivals and Departures separate to stop the potential of cut and past issues meaning that the departure logic uses the arrivals availability

Comment (jackheslop96) [reply test/views/WhatDoYouWantToDoViewSpec.scala:24]: Good idea

**Pull request merged**